includes/defaults.inc.php

```diff
@@ -778,6 +778,7 @@
 $config['poller_modules']['cisco-voice']                 = false;
 $config['poller_modules']['cisco-cbqos']                 = false;
 $config['poller_modules']['cisco-otv']                   = false;
+$config['poller_modules']['cisco-nac']                   = false;
```

Comment:
I think this should just be `nac` as I'm sure cisco won't be the only ones to implement network access control support.

Reply:
Could be, but this module will work only with Cisco switches, because Im using Cisco's MIB.

Reply:
Hi,
The idea would be to actually reuse most of the inside of if but add support for other switches. I am already thinking of my huawei devices on which I run NAC... 
So basically, I would follow laf and suggest to change name to nac or dot1x or whatever is constructor agnostic. 
PipoCanaja